src/library/importer.py

```
@@ -17,14 +24,61 @@
+            dry_run=False,
+            raise_errors=True,
+            use_transactions=True,
+        )
+
+        try:
+            imported_quest = Quest.objects.get(import_id=quest_import_id)
+            imported_quest.visible_to_students = False
+            imported_quest.full_clean()
+            imported_quest.save()
+        except Quest.DoesNotExist():
+            raise ValueError(f"Failed to retrueve imported quest with import_id {quest_import_id}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)